src/libs/actions/App.js

```diff
@@ -121,17 +121,20 @@ function openApp() {
             onyxMethod: CONST.ONYX.METHOD.MERGE,
             key: ONYXKEYS.IS_LOADING_REPORT_DATA,
             value: true,
-        }],
+        },
+        ],
```

Comment:
strange I thought we'd prefer what was there before (`}]`) did the linter complain about this?

Reply:
+1 I think the other way reads better. Same for the others.